Example/src/TagInputExample.js

```diff
@@ -16,18 +19,40 @@ export default class TagInputExample extends Component {
     });
   };
 
+  onChangeHorizontalTags = (horizontalTags) => {
+    this.setState({
+      horizontalTags,
+    });
+  };
+
   labelExtractor = (tag) => tag;
 
+  onParsePendingInput = () => {
+    this.refs.horizontalTagInput.parseTags();
+  }
+
+  onaddCustomTag = () => {
+    this.refs.horizontalTagInput.addCustomTag('Suggested Tag');
```

Comment:
Same comment as above - this shouldn't be accessed via a ref. It would not be necessary if the `text` state were lifted.